content/blog/2025/anchor-position-validity.md

```diff
@@ -226,6 +218,19 @@ bottom layer.
   tab='html,result'
 ) }}
 
+However, if you want to position an element that is inside the popover or
+dialog, you will need to use `position: fixed`. Note that the positioned
+elements are not inside their parents in this example — `position: fixed` moves
+the element's containing block to the viewport and allows positioning to work.
+
+{{ embed.codepen(
+  id='vEBbVXq',
+  title='Top Layer - inside',
+  user='jamessw',
+  height=250,
+  tab='html,result'
+) }}
```

Comment:
Should these embedded demo do anything when I click the buttons in it? It seems like both this demo and the one before only hides the popover stuff when you click open popover, and nothing happens when I click the other button.